nats/aio/client.py

```diff
@@ -1474,13 +1474,20 @@ async def _process_msg(self, sid, subject, reply, data, headers) -> None:
                     sub._pending_size -= payload_size
 
                     await self._error_cb(
-                        SlowConsumerError(subject=subject, sid=sid, sub=sub)
+                        SlowConsumerError(
+                            subject=msg.subject,
+                            reply=msg.reply,
+                            sid=sid,
+                            sub=sub
+                        )
                     )
                     return
                 sub._pending_queue.put_nowait(msg)
             except asyncio.QueueFull:
                 await self._error_cb(
-                    SlowConsumerError(subject=subject, sid=sid, sub=sub)
+                    SlowConsumerError(
+                        subject=msg.subject, reply=msg.reply, sid=sid, sub=sub
```

Comment:
@KMilhan , @wallyqs if that's ok with you, those should be the last changes of this PR. I removed the usage of `Optional` (as well as the import from the `nats.errors` module, because it's not used anywhere else), then applied `yapf`.